src/device-registry/utils/generate-filter.js

```diff
@@ -904,6 +904,15 @@ const generateFilter = {
 
     return filter;
   },
+
+  tips: (request) => {
```

Comment:
filter util function for health tips. We can also be able to specify other fields for querying tips using air quality measurements. @BenjaminSsempala